src/aura/BGE_DataImportBatchEntry/BGE_DataImportBatchEntryController.js

```diff
@@ -63,7 +63,9 @@
      * @description: checks that user has all necessary permissions and then launches modal or displays error
      */
     onEditClick: function(component, event, helper) {
-        helper.checkFieldPermissions(component, event, helper);
+        helper.checkFieldPermissions(component, function () {
+            return helper.openBatchWizard(component);
+        }.bind(helper));
```

Comment:
I know I just used .bind recently, but what do you think about a passing a named function here also?  I've been researching it a bit and it has a few benefits - let me know if you want to discuss

Reply:
Would love to hear more about your thoughts around this and how it'd look